dds/DCPS/RTPS/BaseMessageUtils.cpp

```diff
@@ -4,7 +4,7 @@
  */
 
 #include "BaseMessageUtils.h"
-
+#include "MessageTypes.h"
```

Comment:
There should be a line between them, it's the header corresponding to this cpp file: https://opendds.readthedocs.io/en/master/internal/dev_guidelines.html#includes

```suggestion

#include "MessageTypes.h"

```